app/frontend/entrypoints/pages/duplicateTagsWarning.js

```diff
@@ -0,0 +1,29 @@
+import SCAPE from '@/javascript/lib/global_message_system'
+
+let sendDuplicateTagsWarning = function (plates) {
```

Comment:
Could you rename things from duplicateTags to duplicateTagGroups throughout? e.g. duplicateTagsWarning to duplicateTagGroupsWarning etc.
You aren't actually comparing at the level of individual tags, so it's a bit confusing.